sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.students.controller.getEnrolmentApplicationDetails.e2e-spec.ts

```diff
@@ -31,11 +42,155 @@ describe("ApplicationStudentsController(e2e)-getEnrolmentApplicationDetails", ()
       FakeStudentUsersTypes.FakeStudentUserType1,
       dataSource,
     );
+    application = await saveFakeApplicationDisbursements(
+      db.dataSource,
+      { student },
+      {
+        applicationStatus: ApplicationStatus.Enrolment,
+        createSecondDisbursement: true,
+      },
+    );
+    [firstDisbursement, secondDisbursement] =
+      application.currentAssessment.disbursementSchedules;
+  });
+
+  beforeEach(async () => {
+    // Remove student restriction records associated with the student before each test case.
+    const queryStudentRestrictions = await db.studentRestriction.find({
+      select: { id: true },
+      relations: { student: true },
+      where: {
+        student: { id: student.id },
+      },
+    });
+    db.studentRestriction.remove(queryStudentRestrictions);
+    application.currentAssessment.triggerType =
+      AssessmentTriggerType.OriginalAssessment;
+    await db.application.save(application);
   });
```

Comment:
I am not going to review E2E tests as they are expected on `getCompletedApplicationDetails`. I will review once updated.